### ISSUE TITLE
WM modeling: Tradeoff analysis of event throughput and disk needs given different strategies for processing multi-step workflows in a heterogeneous environment

### ISSUE THREAD

**Issue by khurtado** [dmwm/WMCore#11605]

**Comment by amaltaro** [comment]
@khurtado Kenyi, can you please apply to this - and the other tickets you have created - one of the provided issue templates (for this, the feature request would be the best)? Thanks!

**Comment by khurtado** [comment]
@amaltaro done

**Comment by khurtado** [comment]
@amaltaro It seems we completed all sub-tasks in this issue. Should we close it?

**Comment by khurtado** [comment]
@amaltaro Given the fact that all pending tasks are completed, should we organize another meeting with David Lange? Or, if he was still analyzing the input we provided in the document, should we put this ticket in Waiting until we come back to it for the final part of the issue?

**Comment by amaltaro** [comment]
Yes, I was thinking the same! Maybe the two of us could have a brainstorm on the metrics collected so far and try to have something more solid to discuss with David next week or so(?) In parallel, we can already contact him over email and try to arrange the next meeting (let me get it done).

**Comment by khurtado** [comment]
@amaltaro Thanks! Sounds good to me, let talk about this next week then.

**Comment by amaltaro** [comment]
@khurtado Kenyi, given that this activity is being carried out with David L., I think we should keep it going in Q4 and try to complete it instead of staling it for a quarter. Said that, I am moving it to Q4. But please let me know if you have concerns and/or if we have to move other things further in 2024.
